sdk/plugin/pb/backend.pb.go

```diff
@@ -3769,7 +3779,7 @@ var file_sdk_plugin_pb_backend_proto_rawDesc = []byte{
 	0x6c, 0x5f, 0x64, 0x61, 0x74, 0x61, 0x18, 0x02, 0x20, 0x01, 0x28, 0x09, 0x52, 0x0c, 0x69, 0x6e,
 	0x74, 0x65, 0x72, 0x6e, 0x61, 0x6c, 0x44, 0x61, 0x74, 0x61, 0x12, 0x19, 0x0a, 0x08, 0x6c, 0x65,
 	0x61, 0x73, 0x65, 0x5f, 0x69, 0x64, 0x18, 0x03, 0x20, 0x01, 0x28, 0x09, 0x52, 0x07, 0x6c, 0x65,
-	0x61, 0x73, 0x65, 0x49, 0x64, 0x22, 0xc8, 0x02, 0x0a, 0x08, 0x52, 0x65, 0x73, 0x70, 0x6f, 0x6e,
+	0x61, 0x73, 0x65, 0x49, 0x64, 0x22, 0xe7, 0x02, 0x0a, 0x08, 0x52, 0x65, 0x73, 0x70, 0x6f, 0x6e,
```

Comment:
I don't know why there are a lot of changes in this file, I didn't touch it and I followed the instructions on "protobuf & gRPC generation". If this is cause for concern, let me know.

Reply:
Flyby comment: this looks perfectly normal to me. The protobuf compiler generates lots of diff here anytime the service or message definitions change.

Reply:
Awesome! Thanks for confirming, it puts my mind at ease, haha